src/Sylius/Bundle/AdminBundle/Tests/Command/ChangeAdminUserPasswordCommandTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Tests\Command;
+
+use PHPUnit\Framework\TestCase;
+use Sylius\Bundle\AdminBundle\Command\ChangeAdminUserPasswordCommand;
+use Sylius\Component\Core\Model\AdminUser;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Sylius\Component\User\Security\PasswordUpdaterInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Tester\CommandTester;
+
+final class ChangeAdminUserPasswordCommandTest extends TestCase
+{
+    private const EMAIL = 'sylius@example.com';
+    private const PASSWORD = 'Password';
+
+    private CommandTester $command;
+    private UserRepositoryInterface $userRepository;
+    private PasswordUpdaterInterface $passwordUpdater;
+
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->userRepository = $this->createMock(UserRepositoryInterface::class);
+
+        $this->passwordUpdater = $this->createMock(PasswordUpdaterInterface::class);
+
+        $this->command = new CommandTester(
+            new ChangeAdminUserPasswordCommand($this->userRepository, $this->passwordUpdater),
+        );
+    }
+
+    /** @test */
+    public function it_does_not_reset_password_when_admin_user_is_not_found(): void
+    {
+        $this->userRepository->expects($this->once())
+            ->method('findOneByEmail')
+            ->willReturn(null);
+
+        $this->command->setInputs([
+            'email' => self::EMAIL
+        ]);
+
+        $this->command->execute([]);
+
+        self::assertSame(Command::INVALID, $this->command->getStatusCode());
+    }
+
+    /** @test */
+    public function it_resets_password_for_existing_admin_user(): void
+    {
+        $adminUser = new AdminUser();
+
+        $this->userRepository->expects($this->once())
+            ->method('findOneByEmail')
+            ->willReturn($adminUser);
+        $this->userRepository->expects($this->once())
+            ->method('add')
+            ->with($adminUser);
+
+        $this->passwordUpdater->expects($this->once())
+            ->method('updatePassword')
+            ->with($adminUser);
+
+        $this->command->setInputs([
+            'email' => self::EMAIL,
+            'password' => self::PASSWORD
+        ]);
+
+        $this->command->execute([]);
+
+        self::assertSame(Command::SUCCESS, $this->command->getStatusCode());
```

Comment:
It's a unit test, so it should not be done here - but I'm missing the test that would ensure us that the command **actually** works - so that it's changing the password of the administrator properly. Probably we could write a scenario in the `features/cli/` namespace. Could you give it a try? 🚀

Reply:
Yeah, that was the closest I came up with to test if it "works". Will have a look at the scenario.

That might also be missing the for `CreateAdminUserCommand`, at least the original PR does not contain a scenario.

Reply:
For me, **functional** tests like this one are fine. @Sylius/team & @Sylius/core-team wdyt?